### PR TITLE
L2D15A5

### PR DESCRIPTION
**Pull request by evyryan**
Added question headers.
Fixed fatheads.
One line break error.

### CODE REVIEW

File: kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml [hunk 1]
```diff
@@ -69,13 +69,14 @@
    </p>
             <div xml:id="kzz7yh-f47354-Dd1e122">
                <head xml:id="kzz7yh-f47354-Hd1e124">Quaestio 1</head>
+               <head type="question-title">Utrum deus die septimo non requieuerit<!--TODO word?-->> </head>
```
Comment (RobbieBuhite) [inline kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:72]:
```suggestion
               <head type="question-title">Utrum deus die septimo non requieuerit</head>
```

Comment (RobbieBuhite) [reply kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:72]: "requiesco, -ere, requievi, requietum" - 3rd. sg. perfect active subjunctive form; "to rest"

File: kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml [hunk 2]
```diff
@@ -160,12 +161,13 @@
             </div>
             <div xml:id="kzz7yh-f47354-Dd1e285">
                <head xml:id="kzz7yh-f47354-Hd1e287">Quaestio 2</head>
+               <head type="question-title">Utrum deus diem septimus sanctificauerit</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:164]:
```suggestion
               <head type="question-title">Utrum deus diem septimum sanctificauerit</head>
```

File: kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml [hunk 3]
```diff
@@ -160,12 +161,13 @@
...
                <p xml:id="kzz7yh-f47354-d1e289">
       Quaesio II. 
-      <lb ed="#V" n="106"/>Ecundo queritur vtrum deus diem 
-      <lb ed="#V" n="107"/>
+      <lb ed="#V" n="106"/>SEcundo queritur vtrum deus diem 
+      <lb ed="#V" n="107"/><!--TODO: error line break-->
                   <lb ed="#V" n="108"/>septimum sanctificauerit. Et 
-      <lb ed="#V" n="109"/>videtur quod non: quia qui denon potest recipere gratiam 
+      <lb ed="#V" n="109"/>videtur quod non: quia qui denom potest recipere gratiam
```
Comment (RobbieBuhite) [inline kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:170]: "denom" isn't correct but I am unsure of how to read this line

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

File: kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml (hunk 3 above)
Comment (evyryan) [reply kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:170]: @jeffreycwitt

Comment (jeffreycwitt) [reply kzz7yh-f47354/cod-ve09v2_kzz7yh-f47354.xml:170]: I think this is "quod non". 

I will make this change.

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**